fe/fe-core/src/main/java/com/starrocks/server/GlobalStateMgr.java

```diff
@@ -6796,7 +6796,7 @@ public Future<TStatus> refreshOtherFesTable(TNetworkAddress thriftAddress, Strin
             request.setTable_name(tableName);
             request.setPartitions(partitions);
             try {
-                TRefreshTableResponse response = FrontendServiceProxy.call(thriftAddress, timeout,
+                TRefreshTableResponse response = FrontendServiceProxy.call(thriftAddress, timeout, 3,
```

Comment:
I suggest using a constant integer instead of magic number 3, or even better, make it configurable.